sphinx/developers/matlab-dev.rst

```diff
@@ -23,13 +23,38 @@ MATLAB can be retrieved using::
 
 	java.lang.Runtime.getRuntime.maxMemory
 
-Default |JVM| settings can be increased by Java Heap Memory Preferences of MATLAB
-(R2010a onwards) using
-:menuselection:`Preferences --> General --> Java Heap Memory`.
+Preferences (R2010a+)
+^^^^^^^^^^^^^^^^^^^^^
 
-Alternatively this can also be done by creating a :file:`java.opts` file in
-the startup directory and overriding the default memory settings. We recommend
-using ``-Xmx512m`` in your :file:`java.opts` file. Calling::
+Default |JVM| settings can be increased by `Java Heap Memory Preferences
+<https://mathworks.com/help/matlab/matlab_external/java-heap-memory-preferences.html>`_
+of MATLAB (R2010a onwards) using
+:menuselection:**`Preferences --> General --> Java Heap Memory`**.
+
+See also the next section about :file:`matlab.prf` for the limitation of this method.
+
+matlab.prf
+^^^^^^^^^^
+
+However, the maximum value allowed in the Preferences GUI can still be too small
+for your purpose. In that case, you can directly edit :file:`matlab.prf` file in the
+folder specified by the ``prefdir`` `MATLAB function <https://mathworks.com/help/matlab/ref/prefdir.html>`_
+. Find the parameter ``JavaMemHeapMax`` in the file and increase the number that
+follows the capital ``I`` (in MB) to increase the Java heap memory size.
+
+::
+
+  JavaMemHeapMax=I8800
+
+java.opts
+^^^^^^^^^
+
+Alternatively, this can also be done by creating a :file:`java.opts` file in
+`the startup directory <https://mathworks.com/help/matlab/matlab_env/matlab-startup-folder.html>`_
+ and overriding the default memory settings (see `this page <https://mathworks.com/help/matlab/matlab_env/java-opts-file.html>`_
```

Comment:
The leading space causes the Travis check to fail - see https://travis-ci.org/ome/bio-formats-documentation/jobs/439770938#L1573

```
Warning, treated as error:
/home/travis/build/ome/bio-formats-documentation/target/sphinx/developers/matlab-dev.rst:54:Unexpected indentation.
```